doc/user/content/release-notes.md

```diff
@@ -54,19 +54,40 @@ Wrap your release notes at the 80 character mark.
 
 {{% version-header v0.5.1 %}}
 
-- Default to using a worker thread count equal to half of the machine's
-  physical cores if [`--workers`](/cli/#worker-threads) is not specified.
-- [`TAIL`](/sql/tail) now sends rows instead of being implemented as a
-  `COPY TO` operation. It has gained `timestamp` and `diff` columns.
+- Add the [`COPY TO`](/sql/copy-to) statement, which sends the results of
+  the statement it wraps via the special PostgreSQL [`COPY` protocol][pg-copy].
+
+- Send the rows returned by the [`TAIL`](/sql/tail) statement to the client
+  normally (i.e., as if the rows were returned by a [`SELECT`](/sql/select)
+  statement) rather than via the PostgreSQL [`COPY` protocol][pg-copy]. The new
+  format additionally moves the timestamp and diff information to dedicated
+  `timestamp` and `diff` columns at the beginning of each row.
   **Backwards-incompatible change.**
-- [`COPY TO`](/sql/copy-to) has been added to
-  allow any query to send data via the [Postgres
-  COPY](https://www.postgresql.org/docs/current/sql-copy.html)
-  protocol. This can be used to replace the old `TAIL` behavior.
 
-- Add the `regexp_match` function to search a string with a regular expression.
+  To replicate the old behavior of sending `TAIL` results via the `COPY`
+  protocol, explicitly wrap the `TAIL` statement in a [`COPY TO`](/sql/copy-to)
+  statement.
+
```

Comment:
It might be worth having an example in here, with e.g:

> previously:
> ```sql
> TAIL SELECT * FROM my_view
> ```
> is now:
> ```sql
> COPY (TAIL SELECT * FROM my_view) TO STDOUT
> ```

Reply:
Done.